qiita_pet/handlers/study_handlers/prep_template.py

```diff
@@ -0,0 +1,79 @@
+# -----------------------------------------------------------------------------
```

Comment:
@squirrelo I don't think the tests for this have been added.

One thing that I found useful when I'm addressing comments from a PR is to answer "done" to the comment, so I make sure that I don't forget to address any of the comments.